### ISSUE TITLE
Error when start new VM

### ISSUE THREAD

**Issue by emikg777** [GNS3/gns3-server#1965]
I noticed that I startup new qemu VM(I tested with IOSv & C8000v & CSR100v). It is worked well before update. But now if I create new Qemu VM and start it I receive this:
![image](https://user-images.githubusercontent.com/35261962/132174307-a427fdcb-caff-43a7-97a1-8ac55d929716.png)

Here gns3-server log:
```
Sep 06 11:02:19 gns gns3server[510]: 2021-09-06 11:02:19 INFO qemu_vm.py:1671 logging to /GNS3/projects/b97161e6-48a4-4c0f-90f8-8507af448ac1/project-files/qemu/5b65d148-2daf-4a7b-b5f8-2ac238fb9ff9/qemu-img.log
Sep 06 11:02:19 gns gns3server[510]: 2021-09-06 11:02:19 INFO qemu_vm.py:1673 Executing qemu-img with: /usr/bin/qemu-img check /GNS3/images/QEMU/c8000v-universalk9_8G_serial.17.04.01b.qcow2
Sep 06 11:02:19 gns gns3server[510]: 2021-09-06 11:02:19 INFO qemu_vm.py:1677 /usr/bin/qemu-img returned with 0
Sep 06 11:02:19 gns gns3server[510]: 2021-09-06 11:02:19 INFO qemu_vm.py:1671 logging to /GNS3/projects/b97161e6-48a4-4c0f-90f8-8507af448ac1/project-files/qemu/5b65d148-2daf-4a7b-b5f8-2ac238fb9ff9/qemu-img.log
Sep 06 11:02:19 gns gns3server[510]: 2021-09-06 11:02:19 INFO qemu_vm.py:1673 Executing qemu-img with: /usr/bin/qemu-img create -o backing_file=/GNS3/images/QEMU/c8000v-universalk9_8G_serial.17.04.01b.qcow2 -f qcow2 /GNS3/projects/b97161e6-48a4-4c0f-90f8-8507af448ac1/project-files/qemu/5b65d148-2daf-4a7b-b5f8-2ac238fb9ff9/hda_disk.qcow2
Sep 06 11:02:19 gns gns3server[510]: 2021-09-06 11:02:19 INFO qemu_vm.py:1677 /usr/bin/qemu-img returned with 1
Sep 06 11:02:19 gns gns3server[510]: 2021-09-06 11:02:19 ERROR route.py:217 Node error detected: QemuError
Sep 06 11:02:19 gns gns3server[510]: Traceback (most recent call last):
Sep 06 11:02:19 gns gns3server[510]:   File "/usr/lib/python3.9/site-packages/gns3server/web/route.py", line 198, in control_schema
Sep 06 11:02:19 gns gns3server[510]:     await func(request, response)
Sep 06 11:02:19 gns gns3server[510]:   File "/usr/lib/python3.9/site-packages/gns3server/handlers/api/compute/qemu_handler.py", line 214, in start
Sep 06 11:02:19 gns gns3server[510]:     await vm.start()
Sep 06 11:02:19 gns gns3server[510]:   File "/usr/lib/python3.9/site-packages/gns3server/compute/qemu/qemu_vm.py", line 1057, in start
Sep 06 11:02:19 gns gns3server[510]:     command = await self._build_command()
Sep 06 11:02:19 gns gns3server[510]:   File "/usr/lib/python3.9/site-packages/gns3server/compute/qemu/qemu_vm.py", line 2255, in _build_command
Sep 06 11:02:19 gns gns3server[510]:     command.extend((await self._disk_options()))
Sep 06 11:02:19 gns gns3server[510]:   File "/usr/lib/python3.9/site-packages/gns3server/compute/qemu/qemu_vm.py", line 1881, in _disk_options
Sep 06 11:02:19 gns gns3server[510]:     await self._create_linked_clone(disk_name, disk_image, disk)
Sep 06 11:02:19 gns gns3server[510]:   File "/usr/lib/python3.9/site-packages/gns3server/compute/qemu/qemu_vm.py", line 1705, in _create_linked_clone
Sep 06 11:02:19 gns gns3server[510]:     raise QemuError("Could not create '{}' disk image: qemu-img returned with {}\n{}".format(disk_name,
Sep 06 11:02:19 gns gns3server[510]: gns3server.compute.qemu.qemu_error.QemuError: Could not create 'hda' disk image: qemu-img returned with 1
Sep 06 11:02:19 gns gns3server[510]: qemu-img: /GNS3/projects/b97161e6-48a4-4c0f-90f8-8507af448ac1/project-files/qemu/5b65d148-2daf-4a7b-b5f8-2ac238fb9ff9/hda_disk.qcow2: Backing file specified without backing format
Sep 06 11:02:19 gns gns3server[510]: Detected format of qcow2.

```
IOU works well in new version. Old VM that I started before at most one time same work well. But in old project VM that I not started it gives me same error

**Comment by emikg777** [comment]
Additional information:
I have latest qemu version: 6.1.0
And connect to remote server with Archlinux. I have same behavior with web-gui and gns3-gui(on windows) so it isn't problem gui it is gns3-server problem. I noticed that I have distro 1.5.0 version and updated to newer version 1.6.0 but it haven't any effect on error.

**Comment by grossmj** [comment]
This issue has recently been fixed: https://github.com/GNS3/gns3-server/issues/1955

The problem comes with Qemu 6.1, so you will have to downgrade Qemu or wait that we release v2.2.25

**Comment by grossmj** [comment]
@emikg777 the issue have been fixed in version v2.2.25. Please comment back if you still face the problem.
